packages/php-platform/.github/workflows/build-tagged-version.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+    push:
+        tags:
+            - '**'
+    workflow_dispatch:
```

Comment:
I think you do not want to build this manually as you are missing TAG here

Reply:
🤔  I wanted to be able to manually run the workflow for the already existing tag to trigger the rebuild of the image (for example when some security issue is found and fixed in the base image).

Should it be written differently?